spec.bs

```diff
@@ -947,11 +959,11 @@ A <dfn for=fencedframetype>destination event</dfn> is either a
        beacons](https://wicg.github.io/attribution-reporting-api/#ref-for-request-service-workers-mode).
 
     : [=request/origin=]
-    :: <span class=XXX>Get the origin from somewhere, like the [implementation
-       does](https://source.chromium.org/chromium/chromium/src/+/refs/heads/main:content/browser/fenced_frame/fenced_frame_reporter.cc;l=384;drc=183dd920ea5d6aed92e007c40e226f1f840c0567)</span>.
+    :: |request initiator| if |event| is a [=fencedframetype/destination URL event=],
+       |destination info|'s [=reporting destination info/reporting url declarer origin=] otherwise.
```

Comment:
What about when this origin is null?

Reply:
Resolving since `reporting url declarer origin` is now just an origin, and can't be set to null.